### DISCUSSION TITLE
Message read indicator good or bad?

### DISCUSSION THREAD

**Discussion by Oshyan** [PullDating/Org#120] [Ideas & Features]
One of the things that some existing apps make you pay for is the ability to see whether your message to someone has been read or not ("read receipt indicator"). We are very likely not limiting features like this based on paying or not paying, but I'd like to discuss whether this feature is actually good or bad for people on both sides of the match, their mental health and healthy interaction with the app, and the potential quality of their interaction, etc.

On the one hand read receipts are a potentially useful piece of information that helps solve an issue with existing apps: you often don't get response from someone and you wonder whether you should wait, message them again, or unmatch. If you can see that they read your message but didn't respond, maybe that's an indicator they're not interested (although even that is debatable). However I tend to feel that on balance read receipts are a little questionable, and not a good solution to the problem(s) that exist that they might help address.

In our model there are other factors that *should* help address similar issues, as well as if not better. Limiting concurrent matches #85 is one of those things that should, in theory, have a strong effect to reduce ghosting and ensure that if you are still in someone's match set, they are reasonably interested in you. The "last active" indicator #79 is also something I strongly support, and should help address the "did this person possibly see my message" question without being too invasive of privacy, etc. 

That said I'm interested in hearing whether other people disagree and what reasons a "read receipt" indicator might actually be good.

**Comment by WilliamDormer** [comment]
I'm personally very in favour of having read recepits. Maybe take it to the survey.

**Reply by Oshyan** [reply]
Cool, can you explain why? I mean as a dater *I* definitely want to have it for other people, and would be happy to accept other people being able to know my "read status" in exchange. But that doesn't mean it's the "right" thing objectively speaking for good matches, good user behavior, etc. We often want things that are not necessarily good for us, or for others. I'm just trying to avoid the *presumption* that it's good...

**Reply by WilliamDormer** [reply]
I think that it allows a user to determine if an inactive match is ghosting or simply hasn't read it so that they can find a new match.

**Comment by ajbott** [comment]
I think this should be based on if the user is comfortable with read receipts or not. We don't want to aim to make users uncomfortable, but to give users their own control to make them feel comfortable. If read receipts make them more comfortable, then they should have the option to specify this.
